app/views/videos/index.html.haml

```diff
@@ -1,9 +1,9 @@
 - @categories.each do |category|
-  %article.video_category
+  %article.video_category(id="#{category.name.downcase}")
```

Comment:
ah right, I added that before to help grab the correct tag in my code with JS.  I ended up not using it and forgot to remove it.  Thanks for letting me know!